create-quasar/templates/app/quasar-v2/ts-vite/BASE/README.md

```diff
@@ -3,39 +3,48 @@
 <%= description %>
 
 ## Install the dependencies
+
 ```bash
 yarn
 # or
 npm install
 ```
 
 ### Start the app in development mode (hot-code reloading, error reporting, etc.)
+
 ```bash
 quasar dev
 ```
+
```

Comment:
This line is extra as the tool is not aware of the templating stuff, it perceives it as plain text.

Reply:
Sure will do ASAP. I also use Prettier, but because it was this `markdownlint` extension which warned me, I used it.
But I prefer Prettier. More reliable.

Will add that empty line as well.